apps/sms/views/conversation/test/unit/conversation_test.js

```diff
@@ -143,6 +143,14 @@ suite('conversation.js >', function() {
       id: id,
       participants: participants
     });
+
+    ConversationView.activeThread = Threads.active;
```

Comment:
what we really need (not in this patch though) is to properly call beforeEnter()/afterEnter() so that the view is in a correct state.